File: resourcespoet/src/test/java/com/commit451/resourcespoet/StyleTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class StyleTest {

    @Test
    public void styleTest() throws Exception {
        ArrayList<StyleItem> styleItems = new ArrayList<>();
        styleItems.add(new StyleItem("android:windowBackground", "@android:color/white"));
        styleItems.add(new StyleItem("colorPrimaryDark", "@android:color/black"));
        ResourcesPoet poet = ResourcesPoet.create()
                .addStyle("AppTheme.Dark", "Base.AppTheme.Dark", styleItems);

        TestUtil.assertEquals("style.xml", poet);
    }
}

File: resourcespoet/src/test/java/com/commit451/resourcespoet/StyleTest.kt
package com.commit451.resourcespoet

import org.junit.Test
import java.util.*

/**
 * Tests the resources creation
 */
class StyleTest {

    @Test
    fun styleTest() {
        val styleItems = ArrayList<StyleItem>()
        styleItems.add(StyleItem("android:windowBackground", "@android:color/white"))
        styleItems.add(StyleItem("colorPrimaryDark", "@android:color/black"))
        val poet = ResourcesPoet.create()
                .addStyle("AppTheme.Dark", "Base.AppTheme.Dark", styleItems)

        TestUtil.assertEquals("style.xml", poet)
    }
}
